plugins/modules/vmware_guest_serial_port.py

```diff
@@ -65,12 +66,15 @@
                 In this case, the host name part of the URI should be empty, or it should specify the address of the local host.
             If you use the virtual machine as a client, the URI identifies the remote system on the network.'
       - ' - C(endpoint) (str): Required when I(backing_type=pipe).
-            When you use serial port pipe backing to connect a virtual machine to another process, you must define the endpoints.'
-      - ' - C(no_rx_loss) (bool): Required when I(backing_type=pipe).
-            Enables optimized data transfer over the pipe.
+            When you use serial port pipe backing to connect a virtual machine to another process, you must define the endpoints.
             - choices:
               - client
               - server'
+      - ' - C(no_rx_loss) (bool): Required when I(backing_type=pipe).
+            Enables optimized data transfer over the pipe.
```

Comment:
VMware's default seems to be `False`. Can we set it `False` by default too?

Reply:
Thank you for your comment!  
I'll work on it :)

Reply:
I changed the no_rx_loss default value to False.